dspace/config/input-forms.xml

```diff
@@ -107,6 +107,28 @@
        </field>
 
        <field>
+         <dc-schema>elsevier</dc-schema>
+         <dc-element>identifier</dc-element>
+         <dc-qualifier>pii</dc-qualifier>
+         <repeatable>false</repeatable>
+         <label>PII</label>
+         <input-type>onebox</input-type>
+         <hint>Enter the PII for this item.</hint>
+         <required></required>
+       </field>
```

Comment:
Hi Lieven,

This seems like good advice to me, do you agree?

Best regards,

Letitia Mukherjee
Market Development Manager Sharing Platforms
Mob: +31 6 13 98 98 71

From: Tim Donohue [mailto:notifications@github.com]
Sent: 26 January 2016 15:55
To: DSpace/DSpace
Cc: Mukherjee, Letitia (ELS-AMS)
Subject: Re: [DSpace] DS-2877 Import of ScienceDirect metadata including embargo and linking to or embedding of the final version (#1162)


In dspace/config/input-forms.xml<https://github.com/DSpace/DSpace/pull/1162#discussion_r50844261>:

> @@ -107,6 +107,28 @@

>         </field>

>

>         <field>

> +         <dc-schema>elsevier</dc-schema>

> +         <dc-element>identifier</dc-element>

> +         <dc-qualifier>pii</dc-qualifier>

> +         <repeatable>false</repeatable>

> +         <label>PII</label>

> +         <input-type>onebox</input-type>

> +         <hint>Enter the PII for this item.</hint>

> +         <required></required>

> +       </field>

This "elsevier.identifier.pii" input field likely should not be enabled by default (as the ScienceDirect deposit steps are also not enabled by default). So, it should be commented out by default (you could add a comment saying that sites which want to enable the ScienceDirect import may wish to enable this field)

—
Reply to this email directly or view it on GitHub<https://github.com/DSpace/DSpace/pull/1162/files#r50844261>.

________________________________

Elsevier B.V. Registered Office: Radarweg 29, 1043 NX Amsterdam, The Netherlands, Registration No. 33156677, Registered in The Netherlands.